dotcom-rendering/src/web/components/Counts.stories.tsx

```diff
@@ -79,36 +96,152 @@ export const ShareOnly = () => {
+						pageId="/lifeandstyle/2010/jan/25/deborah-orr-parents-jailers-i-loved"
+						format={{ ...format, theme: ArticlePillar.Sport }}
+					/>
+				</div>
+				<div className="meta-number">
+					<CommentCount
+						discussionApiUrl="https://discussion.theguardian.com"
+						shortUrlId="p/3bdii8"
+						format={{ ...format, theme: ArticlePillar.Sport }}
+					/>
+				</div>
+			</Counts>
+			<Counts format={{ ...format, theme: ArticlePillar.Lifestyle }}>
+				<div className="meta-number">
+					<ShareCount
+						ajaxUrl="https://api.nextgen.guardianapps.co.uk"
+						pageId="/lifeandstyle/2010/jan/25/deborah-orr-parents-jailers-i-loved"
+						format={{ ...format, theme: ArticlePillar.Lifestyle }}
 					/>
 				</div>
 				<div className="meta-number">
 					<CommentCount
-						isCommentable={false}
-						commentCount={239}
-						palette={decidePalette(format)}
+						discussionApiUrl="https://discussion.theguardian.com"
+						shortUrlId="p/3bdii8"
+						format={{ ...format, theme: ArticlePillar.Lifestyle }}
+					/>
+				</div>
+			</Counts>
+			<Counts format={{ ...format, theme: ArticlePillar.Opinion }}>
+				<div className="meta-number">
+					<ShareCount
+						ajaxUrl="https://api.nextgen.guardianapps.co.uk"
+						pageId="/lifeandstyle/2010/jan/25/deborah-orr-parents-jailers-i-loved"
+						format={{ ...format, theme: ArticlePillar.Opinion }}
+					/>
+				</div>
+				<div className="meta-number">
+					<CommentCount
+						discussionApiUrl="https://discussion.theguardian.com"
+						shortUrlId="p/3bdii8"
+						format={{ ...format, theme: ArticlePillar.Opinion }}
+					/>
+				</div>
+			</Counts>
+			<Counts format={{ ...format, theme: ArticleSpecial.Labs }}>
+				<div className="meta-number">
+					<ShareCount
+						ajaxUrl="https://api.nextgen.guardianapps.co.uk"
+						pageId="/lifeandstyle/2010/jan/25/deborah-orr-parents-jailers-i-loved"
+						format={{ ...format, theme: ArticleSpecial.Labs }}
+					/>
+				</div>
+				<div className="meta-number">
+					<CommentCount
+						discussionApiUrl="https://discussion.theguardian.com"
+						shortUrlId="p/3bdii8"
+						format={{ ...format, theme: ArticleSpecial.Labs }}
+					/>
+				</div>
+			</Counts>
+			<Counts format={{ ...format, theme: ArticleSpecial.SpecialReport }}>
+				<div className="meta-number">
+					<ShareCount
+						ajaxUrl="https://api.nextgen.guardianapps.co.uk"
+						pageId="/lifeandstyle/2010/jan/25/deborah-orr-parents-jailers-i-loved"
+						format={{
+							...format,
+							theme: ArticleSpecial.SpecialReport,
+						}}
+					/>
+				</div>
+				<div className="meta-number">
+					<CommentCount
+						discussionApiUrl="https://discussion.theguardian.com"
+						shortUrlId="p/3bdii8"
+						format={{
+							...format,
+							theme: ArticleSpecial.SpecialReport,
+						}}
 					/>
 				</div>
 			</Counts>
 		</Container>
 	);
 };
-ShareOnly.story = { name: 'with comments disabled' };
+Themes.story = { name: 'with different themes' };
```

Comment:
Have we lost the `ShareOnly` part of these stories now?

Reply:
I took that story out because it was moot because the `CommentCount` still shows when it is zero but after thinking about it I realised we don't have a story for when the article does not have commenting enabled so I've included that now. This is effectively the same as what `ShareOnly` was